src/features/profile/Profile.tsx

```diff
@@ -100,13 +106,10 @@ const profile: React.FC<Props> = ({
   );
 };
 
-const mapStateToProps = (state: AppState) => ({
-  theme: state.themeReducer.theme,
-});
-
 const mapDispatchToProps = (dispatch: ThunkDispatch<{}, null, Action>) => ({
   logOutHandler: () => dispatch(logoutHandler()),
-  setTheme: (theme: Themes) => dispatch(chooseTheme(theme)),
 });
 
-export const Profile = connect(mapStateToProps, mapDispatchToProps)(profile);
+export const connectedProfile = connect(null, mapDispatchToProps)(profile);
+
+export const Profile = withTheme(connectedProfile);
```

Comment:
hmm maybe we can find better way to use theme in components and implement own hook for theme ...